packages/admin/cms-admin/src/blocks/PixelImageBlock.tsx

```diff
@@ -219,6 +224,22 @@ export const PixelImageBlock: BlockInterface<PixelImageBlockData, ImageBlockStat
                                 </ListItemIcon>
                                 <FormattedMessage id="comet.blocks.image.cropImage" defaultMessage="Crop image" />
                             </MenuItem>
+                            {dependencyMap["DamFile"] && state.damFile?.id && (
+                                <MenuItem
+                                    onClick={async () => {
+                                        // id is checked three lines above
+                                        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                                        const path = await dependencyMap["DamFile"].resolvePath({ apolloClient, id: state.damFile!.id });
+                                        const url = contentScope.match.url + path;
+                                        window.open(url, "_blank");
```

Comment:
Should this really open in a new tab? How about using a real link here?

Reply:
The main use case is: You are editing a page -> you upload a new asset -> you want to add SEO or license info to the asset (without searching it in the DAM). 

The two user flows are:

1)

You click on the link -> the DAM opens in a new tab -> you add the info -> you close the tab -> you are back to editing your page

2)

You click on the link -> you get a "Discard unsaved changes dialog" where you have to save your (potentially unfinished) changes -> the DAM opens in the same tab -> you add the info -> you use the native go back -> you are back to editing your page

I prefer 1 over 2, especially because you are not forced to save your changes

Reply:
Discussed internally: Opening in a new tab is the preferred way.